docs/src/examples/components/SplitButton/Usage/SplitButtonMainOptionChangeExample.shorthand.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react'
+import { Flex, SplitButton } from '@stardust-ui/react'
+
+class SplitButtonExampleShorthand extends React.Component {
```

Comment:
Wouldn't it be more natural (in this example), if the button doesn't get automatically pressed upon selection of another option?